shanoir-ng-studies/src/main/java/org/shanoir/ng/migration/DistantStudiesShanoirService.java

```diff
@@ -0,0 +1,290 @@
+package org.shanoir.ng.migration;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.shanoir.ng.acquisitionequipment.model.AcquisitionEquipment;
+import org.shanoir.ng.center.model.Center;
+import org.shanoir.ng.manufacturermodel.model.Manufacturer;
+import org.shanoir.ng.manufacturermodel.model.ManufacturerModel;
+import org.shanoir.ng.shared.core.model.IdName;
+import org.shanoir.ng.shared.exception.ShanoirException;
+import org.shanoir.ng.shared.migration.DistantKeycloakConfigurationService;
+import org.shanoir.ng.study.dto.StudyDTO;
+import org.shanoir.ng.study.dto.mapper.StudyMapper;
+import org.shanoir.ng.study.model.Study;
+import org.shanoir.ng.study.service.StudyService;
+import org.shanoir.ng.studycenter.StudyCenter;
+import org.shanoir.ng.subject.model.Subject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Service
+public class DistantStudiesShanoirService {
```

Comment:
Dear Jean-Come, all of this code is already in ShanoirUploader and would be a duplication, see
https://github.com/fli-iam/shanoir-ng/blob/develop/shanoir-uploader/src/main/java/org/shanoir/uploader/service/rest/ShanoirUploaderServiceClient.java, maybe it would be interesting to unify these codes. I know this is quite some work, but maybe be worth it.